pandas/core/dtypes/inference.py

```diff
@@ -24,6 +24,22 @@
 is_interval = lib.is_interval
 
 
+def is_zero_dim_array(obj):
+    """
```

Comment:
this already exists

Reply:
we have ``item_from_zerodim``. so need to reconcile this. you generally don't need to actually check, rather you just unbox.

Reply:
where?

Reply:
```
In [4]: pd.api.types.is_scalar(np.int64(1))
Out[4]: True
```